### DISCUSSION TITLE
Does CustomTkinter have a dropdown menu widget?

### DISCUSSION THREAD

**Discussion by JaiCode08** [TomSchimansky/CustomTkinter#306] [Q&A] [answered]
Does CustomTkinter have a dropdown menu widget? In CustomTkinter/customtkinter/widgets/ there is a dropdown_menu.py but is there a way I can use it?

**Comment by kelltom** [comment, marked as answer]
create a CTKOptionMenu

https://github.com/TomSchimansky/CustomTkinter/wiki/CTkOptionMenu

**Reply by JaiCode08** [reply]
It gave me an error before that customtkinter does not have the attribute "CTkOptionMenu" but I reinstalled it and it works now. Thanks!
